integration/container/mounts_linux_test.go

```diff
@@ -608,15 +608,15 @@ func TestContainerBindMountRecursivelyReadOnly(t *testing.T) {
 	apiClient := testEnv.APIClient()
 
 	containers := []string{
-		container.Run(ctx, t, apiClient, container.WithMount(ro), container.WithCmd(roVerifier...)),
-		container.Run(ctx, t, apiClient, container.WithBindRaw(roAsStr), container.WithCmd(roVerifier...)),
+		container.Run(ctx, t, apiClient, container.WithMount(ro), container.WithCmd(roVerifier...), container.WithSecurityOpt("label=disable")),
+		container.Run(ctx, t, apiClient, container.WithBindRaw(roAsStr), container.WithCmd(roVerifier...), container.WithSecurityOpt("label=disable")),
```

Comment:
I haven't checked, but is this for all tests that use `container.WithMount` / `container.WithBindRaw`, or something specific to these tests?


(considering if there's something global to set up and/or for the test-utility to set it automatically